src/app/pages/abandoned-animal/create-abandoned-animal/create-abandoned-animal.page.html

```diff
@@ -0,0 +1,117 @@
+          <mat-option value="Cachorro (menor a 6 meses)">Cachorro (menor a 6 meses)</mat-option>
+          <mat-option value="Joven (6 meses – 2 años)">Joven (6 meses – 2 años)</mat-option>
+          <mat-option value="Adulto (2 a 7 años)">Adulto (2 a 7 años)</mat-option>
+          <mat-option value="Senior (más de 7 años)">Senior (más de 7 años)</mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
+
+    <!-- Physical -->
+    <div class="col-md-4 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Condición física</mat-label>
+        <mat-select formControlName="physicalCondition">
+          <mat-option value="Saludable">Saludable</mat-option>
+          <mat-option value="Desnutrido">Desnutrido</mat-option>
+          <mat-option value="Herido">Herido</mat-option>
+          <mat-option value="Enfermo">Enfermo</mat-option>
+          <mat-option value="Muy delgado">Muy delgado</mat-option>
+          <mat-option value="Con parásitos visibles">Con parásitos visibles</mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
+
+    <!-- Behavior -->
+    <div class="col-md-4 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Comportamiento</mat-label>
+        <mat-select formControlName="behavior">
+          <mat-option value="Acostado">Acostado</mat-option>
+          <mat-option value="De pie">De pie</mat-option>
+          <mat-option value="Sentado">Sentado</mat-option>
+          <mat-option value="Caminando">Caminando</mat-option>
+          <mat-option value="Asustado">Asustado</mat-option>
+          <mat-option value="Agresivo">Agresivo</mat-option>
+          <mat-option value="Descansando">Descansando</mat-option>
+        </mat-select>
+      </mat-form-field>
+    </div>
+
+    <div class="col-md-4 mb-3">
+      <app-location-form
+        [cantonsList]="cantonHttpService.cantonList()"
+        [districtsList]="cantonHttpService.districtsByCanton()"
+        [neighbourhoodsList]="districtHttpService.neighborhoodsByDistrict()"
+        (selectedCanton)="cantonHttpService.getDistrictsByCantonId($event.id!)"
+        (selectedDistrict)="districtHttpService.getNeighborhoodsByDistrictId($event.id!)"
+        [form]="form"
+        [i18nService]="i18nService"
+        [formsService]="formsService">
+      </app-location-form>
+    </div>
+
+    <!-- observations -->
+    <div class="col-md-12 mb-3">
+      <mat-form-field appearance="outline" class="w-100">
+        <mat-label>Observaciones</mat-label>
+        <textarea matInput rows="3" formControlName="observations"></textarea>
+      </mat-form-field>
+    </div>
+
+    <!-- Upload img -->
+    <div class="col-md-12 mb-3">
+      <label><strong>Subir Imagen (opcional):</strong></label>
+      <input type="file" accept="image/*" (change)="handleImageUpload($event)">
+    </div>
+
+    <!-- IMG preview -->
+    <div class="col-md-12 mb-3" *ngIf="imagePreview">
```

Comment:
este deberia ser mas bien con la sintaxis moderna 


```suggestion
@if(imagePreview) {
 <div class="col-md-12 mb-3">
}
```